### PR TITLE
Add Mass function wrapper to O2DatabasePDG service

### PR DESCRIPTION
**Pull request by benedikt-voelkel**

### CODE REVIEW

**Comment by benedikt-voelkel** [comment]
Thanks to @ktf this is what we came up with to offer a particle mass interface in the `O2DatabasePDG` service

**Review by sawenzel — DISMISSED** [review]

File: Framework/PhysicsSupport/include/Framework/O2DatabasePDGPlugin.h [hunk 1]
```diff
@@ -12,11 +12,21 @@
 #define O2_FRAMEWORK_O2DATABASEPDG_H_
 
 #include "Framework/Plugins.h"
+#include "SimulationDataFormat/O2DatabasePDG.h"
```
Comment (ktf) [inline Framework/PhysicsSupport/include/Framework/O2DatabasePDGPlugin.h]: I suspect the issue is that we should hide this in the .cc.

Comment (benedikt-voelkel) [reply Framework/PhysicsSupport/include/Framework/O2DatabasePDGPlugin.h]: Thanks, your are right. Done

**Comment by ktf** [comment]
Failure unrelated. Merging.

**Pull request merged**